kentikapi/client.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"time"
 
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
```

Comment:
Unless the run-on unreadable words (like `grpcsynthetics`) are some kind of "standard" in the go-land, I'd prefer to use some delimiters.

Reply:
Going by established packages in Go, like `jsoniter` or `retryablehttp` I think the change Daniel suggested is the proper way in Golang.

Reply:
Yes, the idiomatic Go way (TM) is to melt everything into one word. But it might be cargo-culting this sentence:
> By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps.
https://go.dev/doc/effective_go#package-names

Reply:
Done